### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN026.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#72] [General]
# https://blog.3place.cn/3place/lmyn/LMYN026.html

https://blog.3place.cn/3place/lmyn/LMYN026.html

**Comment by picklecai** [comment]
你看这个难以近身的女怪，
她会施展出各种的神通。
我今天一定要让她变，
变成没有耳鼻的妖精。 （1.25.11）

我无意去把她杀掉，
因为她是一个女人。
打击她的威风和行动，
这就是我的用心。” （1.25.12）

罗摩的本意只是打压一下对方威风，不再做坏事就行了。
结果是:

他只用箭当胸一射，
她就倒死在尘埃中。 （1.25.14）

就像浮士德以为梅菲斯特只是让玛格蕾特的妈妈多睡一会儿，只是让海滩上的老夫妇挪一个地方。实际结果却是：玛格蕾特的妈妈直接睡死了，老夫妇直接被打死了。

**Comment by picklecai** [comment]
生主俱利舍湿婆的儿子们，
他们是真正地英勇无敌；
他们都有苦行的力量，
婆罗门！都交到罗摩手里。 （1.25.18）

婆罗门呀！他追随着你，
他最适宜接受这些武器；
神仙们还有艰巨的任务，
要靠这个王子去尽力。” （1.25.19）

此战罗摩掉装备了：得到了所有武器的力量。
